src/test/java/com/stripe/BaseStripeTest.java

```diff
@@ -71,7 +71,7 @@ public static void checkStripeMock() throws Exception {
     }
 
     String version = conn.getHeaderField("Stripe-Mock-Version");
-    if ((version != "master") && (compareVersions(version, MOCK_MINIMUM_VERSION) > 0)) {
+    if (!(version.equals("master")) && (compareVersions(version, MOCK_MINIMUM_VERSION) > 0)) {
```

Comment:
Nice that it caught this! I'd feel a lot safer with this on because this is the type of mistake I make in Java all the time as an infrequent user.